### PR TITLE
Feature/zen 33996: Fix adding events with an empty summary

### PR DESCRIPTION
**Pull request by d-terletskyi**

### CODE REVIEW

**Review by Deer-WarLord — APPROVED** [review]

File: Products/ZenRRD/zencommand.py [hunk 1]
```diff
@@ -502,30 +526,24 @@ def doTask(self):
...
-            self._eventService.sendEvent(
-                STATUS_EVENT,
-                device=self._devId,
-                summary=e.message,
-                component=COLLECTOR_NAME,
-                severity=Error,
+        except defer.CancelledError:
+            message = "Twisted deferred was cancelled."
+            log.error(
+                "Task paused  device=%s ip=%s interval=%s error=%s",
```
Comment (jpeacock-zenoss) [inline Products/ZenRRD/zencommand.py:532]: This message is not correct.  There is no "pausing" occurring in this exception handler.

File: Products/ZenRRD/zencommand.py [hunk 2]
```diff
@@ -502,30 +526,24 @@ def doTask(self):
...
-                log.exception(err_msg)
-            else:
-                log.error(err_msg)
-            self._eventService.sendEvent(
-                STATUS_EVENT,
-                device=self._devId,
-                summary=e.message,
-                component=COLLECTOR_NAME,
-                severity=Error,
+        except defer.CancelledError:
```
Comment (jpeacock-zenoss) [inline Products/ZenRRD/zencommand.py:529]: I'm not convinced that just catching CancelledError is a complete fix.  We don't know why the deferred was cancelled.  Simply logging "deferred was cancelled" provides no useful information to whoever is troubleshooting the message.

**Review by jpeacock-zenoss — CHANGES_REQUESTED** [review]

**Comment by d-terletskyi** [comment]
The fix was changed according to the remarks noted above.

**Comment by d-terletskyi** [comment]
The fix was changed according to the remarks noted above.

**Review by jpeacock-zenoss — APPROVED** [review]

**Pull request merged**